src/test/java/org/mitre/inferno/EndpointTest.java

```diff
@@ -0,0 +1,182 @@
+package org.mitre.inferno;
+
+import org.mitre.inferno.App;
+import org.mitre.inferno.Validator;
+import org.mitre.inferno.rest.IgResponse;
+
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonParser;
+import org.junit.jupiter.api.BeforeAll;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import io.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Represents a tester for api endpoints.
+ */
+public class EndpointTest {
+    private Logger logger = LoggerFactory.getLogger(App.class);
+    private static Validator validator;
+
+    @BeforeAll
+    public static void setup() throws Exception {
+        // get validation results directly from a validator instance
+        try {
+            validator = new Validator("./igs");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        // start the wrapper app
+        String [] args = {};
+        App.main(args);
+        // set up restAssured for tests that center its use
+        String port = System.getProperty("server.port");
+        if (port == null) {
+            RestAssured.port = Integer.valueOf(4567);
+        }
+        else{
+            RestAssured.port = Integer.valueOf(port);
+        }
+        String baseHost = System.getProperty("server.host");
+        if(baseHost==null){
+            baseHost = "http://localhost";
+        }
+        RestAssured.baseURI = baseHost;
+    }
+    
+    @Test
+    public void getResourceTest() throws Exception{
+        try {
+            // api and validator responses
```

Comment:
These comments really aren't necessary. Setup, execute, assert is such a common test pattern you can just separate the sections with a newline.